### PR TITLE
fix: add additional options to pymeilisearch

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Add stop urls as sphinx options

### CODE REVIEW

File: doc-deploy-index/action.yml [hunk 1]
```diff
@@ -75,10 +75,10 @@ inputs:
...
     type: string
 
-  pymeilisearchopts:
+  stop-urls:
     description: >
-      Set of options to pass to `pymeilisearch <https://pymeilisearch.docs.ansys.com/version/stable/>`_
-      while scraping. Default is ``None``.
+      A list of stop points when scraping URLs. If specified,
+      crawling will stop when encountering any URL containing any of the strings in this list.
     default: ''
```
Comment (RobPasMue) [inline doc-deploy-index/action.yml:82]: Wouldn't it be better to just allow all the peimiliesarch possible options like it was done previously?

**Review by RobPasMue — CHANGES_REQUESTED** [review]
LGTM but I have my concerns... see comment

File: doc-deploy-index/action.yml (hunk 1 above)
Comment (jorgepiloto) [reply doc-deploy-index/action.yml:82]: You are right, Roberto. I did not knew this input was available. It looks like it is not being used in the final command.

Let us go with this input and enable it in the command, @Revathyvenugopal162.

Comment (Revathyvenugopal162) [reply doc-deploy-index/action.yml]: Sure revert as options and added it in command

**Review by RobPasMue — APPROVED** [review]
Way better now! :)

**Comment by jorgepiloto** [comment]
I will cherry-pick the latest two fixes and perform a new patch release.

**Pull request merged**